doc/manual/installation/prerequisites-source.xml

```diff
@@ -77,6 +87,15 @@
   modify the parser or when you are building from the Git
   repository.</para></listitem>
 
+  <listitem><para>The <literal>libseccomp</literal> is used to provide
```

Comment:
I didn't realise this was optional. I just assumed it was required because configure bombs out if you don't have it.